test/format/config.c

```diff
@@ -938,6 +955,93 @@ config_lsm_reset(TABLE *table)
     }
 }
 
+/*
+ * config_mirrors --
+ *     Configure table mirroring.
+ */
+static void
+config_mirrors(void)
+{
+    u_int i, mirrors;
+    char buf[100];
+    bool already_set, explicit_mirror;
+
+    /* Check for a CONFIG file that's already set up for mirroring. */
+    for (already_set = false, i = 1; i <= ntables; ++i)
+        if (NTV(tables[i], RUNS_MIRROR)) {
+            already_set = tables[i]->mirror = true;
+            if (g.base_mirror == NULL && tables[i]->type != FIX)
+                g.base_mirror = tables[i];
+        }
+    if (already_set) {
+        if (g.base_mirror == NULL)
+            testutil_die(EINVAL, "no table configured that can act as the base mirror");
+        return;
+    }
+
+    /*
+     * Mirror configuration is potentially confusing: it's a per-table configuration (because it has
+     * to be set for subsequent runs so we can tell which tables are part of the mirror group), but
+     * it's configured on global basis, causing the random selection of a group of tables for the
```

Comment:
Nitpick: "on global basis" -> "on a global basis"

Reply:
[WT-9653](https://jira.mongodb.org/browse/WT-9653) will fix this.